Source/Applications/SystemCenter/wwwroot/Scripts/TSX/SystemCenter/ValueListGroup/ValueListForm.tsx

```diff
@@ -43,14 +43,17 @@ export default function ValueListForm(props: IProps) {
             e.push('Value must be less than 200 characters.');
         if (props.Record.AltValue != null && props.Record.AltValue.length > 200)
             e.push('Label must be less than 200 characters.');
-        
-    }, [props.Record])
+
+        props.SetErrors(e);
+    }, [props.Record, props.SetErrors])
 
     function Valid(field: keyof (SystemCenter.Types.ValueListItem)): boolean {
         if (field == 'Value')
             return props.Record.Value != null && props.Record.Value.length > 0 && props.Record.Value.length <= 200;
         else if (field == 'AltValue')
             return props.Record.AltValue == null || props.Record.AltValue.length <= 200;
+        else if (field == 'SortOrder')
+            return props.Record.SortOrder != null && props.Record.SortOrder >= 0
```

Comment:
Out of curiosity is there a reason we limit `SortOrder` to be positive?

Reply:
No particular reason, I didn't see the need of using negative numbers, but I can get rid of the positive check as we do allow negatives.

Reply:
You do need to restrict this to whole numbers only tho. `gpa-gemstone` has a `isInteger` function you can use (it will work for negative numbers too.